HISTORY.rst

```diff
@@ -0,0 +1,10 @@
+.. :changelog:
```

Comment:
I don't want this yet... it's changing too much and doesn't even have an official public API yet, so I don't wanna worry about updating a changelog